utils.js

```diff
@@ -1,20 +1,20 @@
-export function hex_to_vec(hex) {
+export function hex_to_rgb(hex) {
   if (hex.charAt(0) === '#') {
```

Comment:
I had an idea in the back of my mind for quite a while right now and this PR looks like a nice place to finally implement it. What do you think about removing this block above, and skip the `#` sign from hex colors we're using? This is already happening in some places (like [this example](https://github.com/michalbe/cervus/blob/master/_example/animation/game.js#L7)).

Reply:
I don't have a strong feeling about this. The `#` is a sure sign that what you're seeing is a color and I would probably lean towards leaving it. `123456` doesn't look like much but `#123456` is quite different. In addition some editors highlight the hex colors with the color they define, which helps readability even more.

Reply:
Ha, even GitHub does this! #bada55.

Reply:
Fair enough.